expertise/service/expertise.py

```diff
@@ -366,17 +360,18 @@ def _prepare_config(self, request, job_id=None, client_v1=None, client=None) ->
             working_dir = self.working_dir
         )
         self.logger.info(f"Config validation passed - {config.to_json()}")
+        return config, or_client.token
```

Comment:
why do you need to run the token here? can you get the token after calling this function?